javaslang-benchmark/src/test/java/javaslang/MemoryUsage.java

```diff
@@ -0,0 +1,36 @@
+package javaslang;
```

Comment:
Actually, for small arrays it's even better :).
Also, I might be able to get rid of a few values at the very end - we'll see.